3_edk_ii_dec_file_format/310_pcd_sections.md

```diff
@@ -128,7 +128,7 @@ PCDs listed in `PcdsFeatureFlag` sections must only be listed in
 <PcdStruct>       ::= <PcdName> <FS> <PtrVal> <FS> {<CName>} {<CArrayName>} <FS> <Token> "{" <EOL>
                       <TS> <PcdStructHF>
                       <TS> <PcdStructPKGs> "}" <CbOrEol>
-<PcdStructHF>     ::= "<HeaderFile>" <EOL> <PcdStructFile>
+<PcdStructHF>     ::= "<HeaderFiles>" <EOL> <PcdStructFile>

```

Comment:
Commit message requires Signed-off-by and Contributed-under tags

See: https://github.com/tianocore-docs/edk2-TemplateSpecification/wiki/TianoCore-Documents-Commit-Message-Format